ProcessLib/CompareJacobiansJacobianAssembler.cpp

```diff
@@ -0,0 +1,424 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "CompareJacobiansJacobianAssembler.h"
+
+#include <sstream>
+
+#include "MathLib/LinAlg/Eigen/EigenMapTools.h"
+
+#include "CreateJacobianAssembler.h"
+
+namespace
+{
+//! Dumps a \c double value as a Python script snippet.
+void dump_py(std::ofstream& fh, std::string const& var, double const val)
+{
+    fh << var << " = " << val << '\n';
+}
+
+//! Dumps a \c std::size_t value as a Python script snippet.
+void dump_py(std::ofstream& fh, std::string const& var, std::size_t const val)
+{
+    fh << var << " = " << val << '\n';
+}
+
+//! Dumps an Eigen vector as a Python script snippet.
```

Comment:
Dumps an arbitrary vector (of type Vec) as a Python script snippet.

Reply:
:white_check_mark: